controllers/post-survey-controller.js

```diff
@@ -0,0 +1,65 @@
+const Sequelize = require('sequelize');
+
+const { Op } = Sequelize;
+
+const PostSurveyController = PostSurvey => {
+  /**
+   *
+   * @param {{}} query
+   * @param {string} accountSid
+   */
+  const getPostSurveys = async (query, accountSid) => {
+    // const {} = query;
+    const queryObject = {
+      limit: 10,
+    };
+
+    queryObject.where = {
+      [Op.and]: [accountSid && { accountSid }],
+    };
+
+    const postSurveys = await PostSurvey.findAll(queryObject);
+    return postSurveys;
+  };
```

Comment:
This is not very helpful at the moment, since it just lists 10 records. I can remove it or leave it for future use (like "list all post survey results").

Reply:
It would be find to delete it.  Or, leave it in only for 'test' or 'development' environments.